tests/ttlsfuzzer

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+# Copyright (C) 2024 Jakub Jelen <jjelen@redhat.com>
+# SPDX-License-Identifier: Apache-2.0
+
+source "${TESTSSRCDIR}/helpers.sh"
+
+if [[ ! -d "${TESTSSRCDIR}/../tlsfuzzer/tlsfuzzer" ]]; then
+    title "TLS fuzzer is not available -- skipping"
+    exit 77;
+fi
+
+TMPFILE="${PWD}/tls-fuzzer.$$.tmp"
+PORT=4433
+PYTHON=$(which python3)
+
+if [[ -f /etc/debian_version ]] && grep Ubuntu /etc/lsb-release; then
+    # the ubuntu builds miss Brainpool curves, but Debian has them already
```

Comment:
Do we actually support Brainpool curves with any of the tokens?

Reply:
No, but this is the list of curves that are announced by the server if I understand it right.